src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -1904,46 +1975,89 @@ public void addRoute(String remoteId, String uuid, int metric) {
     routeTable.addRoute(remoteId, uuid, metric);
   }
 
-  static public ServiceInterface start(String name) {
-    ServiceInterface si = null;
-    try {
-      si = create(name);
-    } catch (Exception e) {
-      Runtime.getInstance().error(e);
-      log.error("create({}) threw", name, e);
-    }
+  static public ServiceInterface start() {
+    return start(null, null, null);
+  }
 
-    if (si != null) {
-      return start(name, si.getType());
-    }
+  static public ServiceInterface start(String name, String type) {
+    return start(null, name, type);
+  }
 
-    return null;
+  static public ServiceInterface start(String name) {
+    return start(null, name, null);
   }
 
-  static public ServiceInterface start(String name, String type) {
+  static public ServiceInterface start(Plan inPlan, String name) {

```

Comment:
why is the argument called "inPlan"  ? why not just "plan"  ?  add javadoc to describe why you've called it "in"Plan

Reply:
this commit - peers needed to send in their plan
removing this ..
there is only "one" plan